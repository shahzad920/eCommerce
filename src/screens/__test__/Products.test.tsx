/**
 * @format
 */

import 'react-native';
import React from 'react';
import { Products } from '..';

jest.mock('node-fetch', () => require('fetch-mock-jest').sandbox())

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('Product List Screen Renders correctly', () => {
   
 const tree = renderer.create(<Products test/>).toJSON();
    expect(tree).toMatchSnapshot();
});
