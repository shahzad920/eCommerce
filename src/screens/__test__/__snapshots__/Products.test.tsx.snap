// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product List Screen Renders correctly 1`] = `
<RCTScrollView
  data={[]}
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={[]}
  testID="Product-List"
  viewabilityConfigCallbackPairs={[]}
>
  <View />
</RCTScrollView>
`;

exports[`renders correctly 1`] = `
<RCTScrollView
  data={[]}
  getItem={[Function]}
  getItemCount={[Function]}
  keyExtractor={[Function]}
  onContentSizeChange={[Function]}
  onLayout={[Function]}
  onMomentumScrollBegin={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={[]}
  testID="Product-List"
  viewabilityConfigCallbackPairs={[]}
>
  <View />
</RCTScrollView>
`;
